packages/hash/api/src/graphql/resolvers/middlewares/canAccessAccount.ts

```diff
@@ -11,7 +11,7 @@ import { ResolverMiddleware } from "./middlewareTypes";
 export const canAccessAccount: ResolverMiddleware<
   GraphQLContext,
   {
-    ownedById: Scalars["ID"];
+    ownedById: Scalars["AccountId"];
```

Comment:
We probably want `OwnedById` here too right?

Reply:
Yes, thank you! 3c9018ae5cc91804923a9ed854b390674f6bfba7